packages/ceramic-core/package.json

```diff
@@ -36,10 +36,12 @@
     "@ethersproject/base64": "^5.0.0-beta.134",
     "@ethersproject/providers": "^5.0.0-beta.144",
     "@ethersproject/random": "^5.0.0-beta.136",
+    "@textile/powergate-client": "^0.2.0",
     "cids": "^0.8.0",
     "cross-fetch": "^3.0.5",
     "did-resolver": "^2.0.1",
     "fast-json-patch": "^3.0.0-1",
+    "ipfs-http-client": "^45.0.0",
```

Comment:
Same thing with this really. We could basically make the classes IpfsPinning and PowergatePinning into their own packages. You could then create an instance of these an pass into the `Ceramic.create` function. This would also allow others to implement their own pinning backends without them having to have support within ceramic-core.

Reply:
@simonovic86 We should probably do something similar for the anchor-service at some point?

Reply:
Yes, this makes sense. We should definitely do the same thing for the anchor service client, as well.